### DISCUSSION TITLE
some errors with /run_tests -j 6

### DISCUSSION THREAD

**Discussion by JOOsna** [idaholab/moose#29086] [Q&A Getting Started]
### Check these boxes if you have followed the posting rules.

- [X] I have consulted the [Posting Guidelines](https://github.com/idaholab/moose/discussions/18270).
- [X] I have searched the [Discussions Forum](https://github.com/idaholab/moose/discussions) and [MOOSE Framework Troubleshooting](https://mooseframework.inl.gov/help/troubleshooting.html) and have not found what I was looking for
- [X] Q&A Getting Started is the most appropriate category for my question (trouble installing, beginner user, ...)

### Issue or question about MOOSE

hello

ive got some errors with /run_tests -j 6 after build moose(make -j 6)

The code below is the corresponding error code

plz help me :(

### (Optional) code in question / simulation log / errors

```shell
controls/web_server_control.control/bool ................................................. FAILED (CRASH)
controls/web_server_control.control/real ................................................. FAILED (CRASH)
controls/web_server_control.control/int .................................................. FAILED (CRASH)
controls/web_server_control.control/string ............................................... FAILED (CRASH)
controls/web_server_control.control/vec_real ............................................. FAILED (CRASH)
controls/web_server_control.control/vec_int .............................................. FAILED (CRASH)
controls/web_server_control.control/vec_string ........................................... FAILED (CRASH)
controls/web_server_control.get_postprocessor ............................................ FAILED (CRASH)
controls/web_server_control.parallel_consistent ............................. [min_cpus=2] FAILED (CRASH)
controls/web_server_control.errors/set_controllable_no_exist ............ FAILED (EXPECTED ERROR MISSING)
controls/web_server_control.errors/postprocessor_no_exist ............... FAILED (EXPECTED ERROR MISSING)
controls/web_server_control.errors/set_controllable_unregistered_type ... FAILED (EXPECTED ERROR MISSING)
controls/web_server_control.errors/set_controllable_bad_convert_json .... FAILED (EXPECTED ERROR MISSING)
controls/web_server_control.errors/set_controllable_vector_non_array .... FAILED (EXPECTED ERROR MISSING)
```

**Comment by GiudGiud** [comment]
Hello

You can ignore these for now. It just means on object in moose in the Controls system won't work for you. 

If you paste the log above this summary of the errors we can try to diagnose why they occurred.

**Reply by JOOsna** [reply]
Thank u :) 

Your answer is very helpful
